src/content/plugins/internal-plugins.md

```diff
@@ -27,7 +27,7 @@ Plugins affecting the environment of the compiler.
 
 `node/NodeEnvironmentPlugin()`
 
-Applies node.js style filesystem to the compiler.
+Applies Node.js style filesystem to the compiler.
```

Comment:
There is 5 occurrences of this typo, didn't wanted to add a comment for each one, as advice, search in the entire document for typos.

Reply:
i think i did fix them all